scripts/in_container/run_prepare_airflow_packages.py

```diff
@@ -79,31 +79,34 @@ def process_summary(success_message: str, error_message: str, completed_process:
 rich.print("[bright_blue]Checking airflow version\n")
 
 airflow_version = subprocess.check_output(
-    [sys.executable, "setup.py", "--version"], text=True, cwd=AIRFLOW_SOURCES_ROOT
+    [sys.executable, "-m", "hatch", "version"], text=True, cwd=AIRFLOW_SOURCES_ROOT
 ).strip()
 
 rich.print(f"[green]Airflow version: {airflow_version}\n")
 
 RELEASED_VERSION_MATCHER = re.compile(r"^\d+\.\d+\.\d+$")
 
-command = [sys.executable, "setup.py"]
+build_command = [sys.executable, "hatch", "build"]
 
 if version_suffix:
     if RELEASED_VERSION_MATCHER.match(airflow_version):
         rich.print(f"[warning]Adding {version_suffix} suffix to the {airflow_version}")
-        command.extend(["egg_info", "--tag-build", version_suffix])
+        sys.exit(2)
```

Comment:
Are you sure you want to exit here? Or is this WIP code? Then the "warning" should be converted to `[red]`

Reply:
It's definitely A WIP

Reply:
This WIP is now complete.